doc/modules/decomposition.rst

```diff
@@ -695,13 +695,22 @@ the mean of all elements of the data), and NNDSVDar (in which the zeros are set
 to random perturbations less than the mean of the data divided by 100) are
 recommended in the dense case.
 
-:class:`NMF` can also be initialized with random non-negative matrices, by
-passing an integer seed or a ``RandomState`` to :attr:`init`.
+:class:`NMF` can also be initialized with correctly scaled random non-negative
+matrices. An integer seed or a ``RandomState`` can also be passed to
```

Comment:
(by setting ``init="random"``)? Otherwise it might sound like the user needs to worry about correctly scaling something somewhere.